charts/distributionplot/src/distributionplot-properties.js

```diff
@@ -318,6 +318,52 @@ export default function propertyDefinition(env) {
 
   const measureAxis = {
     uses: 'axis.picasso.measureAxis',
+    items: {
+      startAt: {
```

Comment:
is the definition of startAt the same in all charts that have it
maybe add it in default-properties in sense-client to support 

```
startAt: {
   use: 'startAt'
}
```

in the charts that needs it

Reply:
start at added to chart that are not bar type, for example bar chart and histogram won't have it. Also the autoMinMax, minMax, min are not in same location, for example combo chart, line chart and scatterplot.